src/xdocs/config.xml

```diff
@@ -988,44 +981,31 @@
 
       <p>
         Checkstyle also strictly enforces the encoding attribute of an
-        XML declaration.  If Checkstyle rejects your configuration
+        XML declaration. If Checkstyle rejects your configuration
         document's encoding, correct the value of the encoding
         attribute, or remove the encoding attribute entirely.
       </p>
 
       <p>
         For a complete example of a configuration XML document, examine
-        file <code>docs/sun_checks.xml</code> that checks the Sun coding
-        conventions.
+        files
+        <a href=
+          "https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/sun_checks.xml">
+          sun_checks.xml
```

Comment:
Shouldn't we use a similar search as on modules links so if we move the file we won't have to remember to update the links?
https://github.com/search?q=path%3Asrc%2Fmain%2Fresources+filename%3Asun_checks.xml+repo%3Acheckstyle%2Fcheckstyle

Reply:
Direct link is more user friendly, I saw a cases when users do not understand how to get to file in such search result.
Change that we will change location is not big.
We used such searches only for content in file, that is very reasonable as as content it changing and direct link to line number will not work.